README.md

```diff
@@ -8,7 +8,7 @@ This implementation currently offers entry points for supporting the following f
 
 - NVIDIA DLSS for Vulkan, by supporting the relevant adapter information by querying from [Vulkan](https://www.vulkan.org/).
 - NVIDIA DLSS for D3D11 and D3D12, by querying from Vulkan and forwarding the relevant calls into DXVK / VKD3D-Proton.
-- NVIDIA Reflex, by forwarding the relevant calls into [LatencyFleX](https://github.com/ishitatsuyuki/LatencyFleX).
+- NVIDIA Reflex, by forwarding the relevant calls into either DXVK / VKD3D-Proton or [LatencyFleX](https://github.com/ishitatsuyuki/LatencyFleX).
```

Comment:
Probably unnecessary detail at this level but I'd add something like

```suggestion
- NVIDIA Reflex, by forwarding the relevant calls into either DXVK / VKD3D-Proton (which in turn rely on Vulkan device having support for VK_NV_low_latency2 extension) or [LatencyFleX](https://github.com/ishitatsuyuki/LatencyFleX).
```

Feel free to ignore me though :see_no_evil:

Reply:
I would prefer that we add those details to https://github.com/jp7677/dxvk-nvapi/wiki/Tips-and-tricks-for-usage-with-DXVK-NVAPI#reflex